shepherd/Utils.py

```diff
@@ -0,0 +1,11 @@
+
+# Values for the cooldowns
+from enum import Enum, unique
+
+@unique # pylint: disable=invalid-name
+class CONSTANTS(Enum):
+    TWO_X_COOLDOWN = 1
```

Comment:
I think these need to map to different values. Not exactly sure how enums work in python to be honest.